docs/api/browser-window.md

```diff
@@ -169,6 +169,15 @@ Emitted when window is minimized.
 
 Emitted when window is restored from minimized state.
 
+### Event: 'resize'
+
+Emitted when window is getting resized on all platforms.
+
+### Event: 'move'
```

Comment:
Agreed, i guess for linux/windows will be separating out these events then. @zcbenz even the observers in [aura](https://code.google.com/p/chromium/codesearch#chromium/src/ui/aura/window_tree_host_observer.h&sq=package:chromium&l=24) provide live move notification rather than on completion. Thoughts ?